js/jquery.bind-first.js

```diff
@@ -6,64 +6,63 @@
  * @license
```

Comment:
I think that we need to ignore that file too from the linting.

Reply:
Yes I think I fixed it at the beginning cause I wasn't sure but maybe we should ignore it...
Can you ignore the modifications for this file, or should I do something?

Reply:
you can do a commit that revert that file to the original that add also an exception to eslint file.

Reply:
I can do a commit to revert that file but to ignore the file I must create an eslintignore file.
https://eslint.org/docs/user-guide/configuring/ignoring-code#the-eslintignore-file
if that's okay with you I'll take care of it right away

Reply:
I think, maybe, with the help of Old Gods, that it's good now...